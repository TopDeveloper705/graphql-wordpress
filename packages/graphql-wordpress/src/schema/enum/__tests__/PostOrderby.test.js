import POST_ORDERBY from 'enum/PostOrderby';

describe('Test enum values', () => {
  test('POST_ORDERBY values', () => {
    expect(POST_ORDERBY.getValues()).toMatchSnapshot();
  });
});
