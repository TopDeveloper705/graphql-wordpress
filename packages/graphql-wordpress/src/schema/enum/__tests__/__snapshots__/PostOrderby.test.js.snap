// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values POST_ORDERBY values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "DATE",
    "value": "date",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "RELEVANCE",
    "value": "relevance",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "ID",
    "value": "id",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "INCLUDE",
    "value": "include",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "TITLE",
    "value": "title",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "SLUG",
    "value": "slug",
  },
]
`;
